handlers/validate-spec/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	"github.com/qri-io/jsonschema"
```

Comment:
I wouldn't allow updating this dependency anymore as we are really expecting a sensitive system to work with this and we cannot afford not noticing a change that has a big impact in our security